corehq/apps/hqcase/management/commands/reindex_sql_forms_in_domain.py

```diff
@@ -0,0 +1,22 @@
+from django.core.management import BaseCommand
+
+from corehq.form_processor.backends.sql.dbaccessors import *
+from corehq.pillows.xform import *
+
+
+def reindex_sql_forms_in_domain(domain):
+    for state, _ in XFormInstanceSQL.STATES:
+        doc_ids = FormAccessorSQL.get_form_ids_in_domain_by_state(domain, state)
+        for form in FormAccessorSQL.get_forms(doc_ids):
+            form_json = form.to_json(include_attachments=True)
+            txform = transform_xform_for_elasticsearch(form_json)
+            reindexer = get_sql_form_reindexer()
```

Comment:
Instead of using the reindexer you could just instantiate the doc processor directly: 
```
doc_processor = BulkPillowReindexProcessor(
    get_es_new(), XFORM_INDEX_INFO, doc_filter=None, doc_transform=transform_xform_for_elasticsearch
)
```